cmd/pmm-managed/main.go

```diff
@@ -237,6 +240,48 @@ func runDebugServer(ctx context.Context) {
 	cancel()
 }
 
+func runTelemetryService(ctx context.Context) {
+	l := logrus.WithField("component", "TELEMETRY")
+
+	disabled := strings.ToLower(os.Getenv("DISABLE_TELEMETRY"))
+	if disabled == "true" || disabled == "1" {
+		l.Infof("Telemetry is disabled by DISABLE_TELEMETRY env var")
+		return
+	}
+
+	cfg, _ := telemetry.LoadConfig(*telemetryConfigF)
+
+	updated, err := telemetry.CheckConfig(cfg)
```

Comment:
If LoadConfig can't load config and return `nil` for cfg, that line will panic. Let's not ignore LoadConfig error. Or combine LoadConfig and CheckConfig into single function – do we need to load invalid configs for something?